Tasks/Grunt/grunttask.ts

```diff
@@ -41,11 +47,85 @@ gt.pathArg(gruntFile);
 
 gt.argString(tl.getInput('arguments', false));
 
-gt.exec()
-.then(function(code) {
-	tl.setResult(tl.TaskResult.Succeeded, tl.loc('GruntReturnCode', code));
-})
-.fail(function(err) {
+var enableCoverage = tl.getBoolInput('enableCodeCoverage');
+
+if (enableCoverage) {
+	var testSrc = tl.getPathInput('testFiles', true, false);
```

Comment:
That's a problem I found out on various sources that running istanbul on windows is a bit tedious and we have to run on windows this way.